src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -264,13 +242,10 @@ class WorkspaceInvitePage extends React.Component {
         if (!this.validate()) {
             return;
         }
-
-        this.setState({shouldDisableButton: true}, () => {
-            const logins = _.map(this.state.selectedOptions, option => option.login);
-            const filteredLogins = _.uniq(_.compact(_.map(logins, login => login.toLowerCase().trim())));
-            Policy.addMembersToWorkspace(filteredLogins, this.state.welcomeNote || this.getWelcomeNote(), this.props.route.params.policyID);
-            Navigation.goBack();
-        });
+        const logins = _.map(this.state.selectedOptions, option => option.login);
+        const filteredLogins = _.uniq(_.compact(_.map(logins, login => login.toLowerCase().trim())));
```

Comment:
Same here. Can we actually extract this into an utils method since we are calling it in two places?